src/components/setup/HostView.vue

```diff
@@ -0,0 +1,66 @@
+<script setup lang="ts">
+import { computed } from 'vue'
+import { useCookies } from '@vueuse/integrations/useCookies'
+import { useRouter } from 'vue-router'
+import { useGameDataStore, useMusicPlayerStore, useSpotifyLibraryStore } from '@/stores'
+import SpotifyLibraryLoading from '@/components/setup/SpotifyLibraryLoading.vue'
+import SpotifyLibraryDisplay from '@/components/setup/SpotifyLibraryDisplay.vue'
+import { Button } from '@/components/ui/button'
+import GameDataForm from '@/components/setup/GameDataForm.vue'
+import type { SelectableAlbum, SelectablePlaylist } from '@/types'
+
+const musicPlayerStore = useMusicPlayerStore()
+const spotifyLibraryStore = useSpotifyLibraryStore()
+const gameDataStore = useGameDataStore()
+const router = useRouter()
+const cookies = useCookies()
+const access_token = cookies.get('access_token')
+
+function handleLoadingFinished(playlists: SelectablePlaylist[], albums: SelectableAlbum[]) {
+  spotifyLibraryStore.playlists = playlists
+  spotifyLibraryStore.albums = albums
+}
+
+async function handleGameStart() {
+  if (!musicPlayerStore.ready)
+    return
+
+  await musicPlayerStore.turnOn()
+  const tracks = await spotifyLibraryStore.getTracksFromSelectedSets(access_token, router)
+  gameDataStore.prepareGame(tracks)
+  router.push({ name: 'round', params: { id: '7734' } })
```

Comment:
i love magic number strings in pull requests

Reply:
Will be deleted when next task is done, not relevant